torchmetrics/regression/psnr.py

```diff
@@ -79,71 +34,16 @@ def __init__(
         dist_sync_on_step: bool = False,
         process_group: Optional[Any] = None,
     ):
+        warn(
+            "This `PSNR` was deprecated in v0.4 and will be removed in v0.5. Use `torchmetrics.image.psnr` instead.",
```

Comment:
maybe mention that it was moved? otherwise they might think something was wrong with PSNR, but it's actually the same implementation.